packages/controller-config/src/resources/controller.ts

```diff
@@ -41,6 +42,10 @@ export function ControllerResources({
 
   const controllerCmdlineArgs = [`${opstraceClusterName}`];
 
+  // create dockerhub credentials secret if we have the credentials set
+  // as environment variables
+  collection.add(dockerHubCredsSecret(kubeConfig));
```

Comment:
Is this how the installer after all injects this secret into the k8s cluster?

Reply:
yip, same time as it creates the controller secrets, roles, rolebindings, deployment etc